messenger/make.py

```diff
@@ -67,9 +76,55 @@ def build_octave():
     do_build(make_cmd, 'messenger.mex')
 
 
+def which_matlab():
```

Comment:
`pexept` assumes the binaries are in the path since it replicates BSD utils `which`.

However, `get_matlab_bin` should return absolute path of the matlab binary regardless of if it is in the path or not.

https://github.com/jjangsangy/python-matlab-bridge/blob/master/messenger/make.py#L206

Then running `get_matlab_env` with the path to the binary will also return a dictionary containing that particular configuration environment variables.

https://github.com/jjangsangy/python-matlab-bridge/blob/master/messenger/settings.py#L15